Formula/arangodb.rb

```diff
@@ -11,81 +12,98 @@ class Arangodb < Formula
     sha256 "60d59e09f4a440cffeb077f1de14784519223e6f25ccaaf6df692dc63fd6cac8" => :mavericks
   end
 
-  head do
-    url "https://github.com/arangodb/arangodb.git", :branch => "unstable"
-    depends_on "cmake" => :build
-  end
-
+  depends_on "cmake" => :build
   depends_on "go" => :build
   depends_on "openssl"
 
   needs :cxx11
 
+  # malloc_usable_size is not supported
+  patch :DATA
```

Comment:
If this has already been upstreamed you'll need to move it inside a `stable do` block, otherwise it'll presumably fail to apply for `head`.